src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -792,6 +802,7 @@ export class BaseEditCourseForm extends React.Component {
           {this.getAddCourseRunButton(disabled, pristine, uuid)}
           {editable &&
             <ButtonToolbar className="mt-3">
+              {submitState === 'default' ? cancelButton : null}
```

Comment:
I would double check with Carol, but it feels like `cancelButton` should always be available, unless the form is submitting (Similar to how the save button works `disabled={disabled || submitting}`)

This would allow the user to click 'Cancel' and return to the CourseTable page, regardless of whether they've made a change in the form.